src/Orc.Controls/Orc.Controls.csproj

```diff
@@ -46,10 +53,32 @@
     <EmbeddedResource Include="**/GeneratorParameters.json" />
   </ItemGroup>
 
+  <ItemGroup>
+    <XAMLCombineItems Include="Themes\Geometry.xaml">
+      <TargetFile>Themes/Generic.generated.xaml</TargetFile>
+      <Visible>True</Visible>
+    </XAMLCombineItems>
+    <XAMLCombineItems Include="Themes\Theming.defaultstyles.orccontrols.xaml">
+      <TargetFile>Themes/Generic.generated.xaml</TargetFile>
+      <Visible>True</Visible>
+    </XAMLCombineItems>
+    <XAMLCombineItems Include="Themes\Styles.xaml">
+      <TargetFile>Themes/Generic.generated.xaml</TargetFile>
+      <Visible>True</Visible>
+    </XAMLCombineItems>
+    <XAMLCombineItems Include="Controls\**\*.generic.xaml">
+      <TargetFile>Themes/Generic.generated.xaml</TargetFile>
+      <Visible>True</Visible>
+    </XAMLCombineItems>
+  </ItemGroup>
+
   <ItemGroup>
     <PackageReference Update="NETStandard.Library" Version="2.0.3" />
   </ItemGroup>
 
   <Import Project="$(MSBuildProjectDirectory)\..\Directory.Build.shared.explicit.props" Condition="Exists('$(MSBuildProjectDirectory)\..\Directory.Build.shared.explicit.props')" />
   <Import Project="$(MSBuildProjectDirectory)\..\Directory.Build.shared.mat.props" Condition="Exists('$(MSBuildProjectDirectory)\..\Directory.Build.shared.mat.props')" />
+
+  <Import Project="..\Directory.Build.xamltools.props" />
```

Comment:
See imports above for a bettery way to do this. (check if file exists). We will add these files to the repository template in the future to simplify distribution